### ISSUE TITLE
Always running faield if enable mergestat sync 

### ISSUE THREAD

**Issue by colin-han** [mergestat/mergestat#1177]
Hi,

When I enabled the mergestat sync with some repos, the worker will take up a lot of cpu and will not terminate, even if I disable the sync. 

When I looking at the `sqld.job_logs` table, I found the following error message:

```
pulling git repository: file:///projects/xxxxxxxx/xxxxxxxxx
finished git clone successfully: file:///projects/xxxxxxxx/xxxxxxxxx
cloned repository to /tmp/mergestat-repo-82d2ff81-b97b-4ae8-8971-8d72d706a4bc-3488584103 and mounting it at /mergestat/repo
{"msg":"exec container process `/syncer/entrypoint.sh`: Invalid argument","level":"error","time":"2024-09-23T06:23:26.021765Z"}
failed to run image: exit status 1
```

How can I fix the issue? 

PS: Git Blame sync has similar issues.

Thanks
Colin

**Comment by robsonke** [comment]
Same issue here, I just cloned the repo, ran docker-compose up and added a public repo plus some default syncs.
Maybe related to using a macbook with arm cpu.

**Comment by nafeger** [comment]
I am seeing this issue as well from a fresh checkout and running on OS X 
![Screenshot 2024-10-31 at 2 44 08 PM](https://github.com/user-attachments/assets/60cee384-fee9-46be-8a4b-437a3f6e0bd6)

ChatGPT seems to think this might have something to do with line endings

**Comment by colin-han** [comment]
> Same issue here, I just cloned the repo, ran docker-compose up and added a public repo plus some default syncs. Maybe related to using a macbook with arm cpu.

Yes, I'm use a macbook too, but for a intel cpu. Seem the issue is only occurred on MacOS.

**Comment by ddrao** [comment]
Any update?

**Comment by brett-larson-prizepicks** [comment]
Any update to this issue?
